src/Applications/LDAS_App/ens_forcing/enpert_forc.csh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/csh -f 
+
+## env : ADAS_EXPDIR,  GRID,  NENS, RUNDIR
```

Comment:
@saraqzhang, this script needs a brief statement of how it fits into GEOSldas and what it does.  Something like:

```
# Script to create re-centered ensemble land forcing (lfo) files, typically used in LADAS:
# 1. Regrids coarse-resolution lfo files from the ADAS atmospheric ensemble to the higher resolution 
#   of the single-member central (or deterministic) simulation.
# 2. Computes perturbations from regridded lfo files.
# 3. Applies perturbations to lfo files from central simulation. 
#
# Requires environment variables ADAS_EXPDIR, GRID, NENS, RUNDIR
```

Reply:
agree.